### PR TITLE
gh-103326: Remove `Python/importlib.h`

### PR DESCRIPTION
**Pull request by sobolevn**
Looks like there are no more references of it 🎉 
(except history)

```
» ag importlib.h
Misc/HISTORY
6777:- Issue #15466: Stop using TYPE_INT64 in marshal, to make importlib.h (and other
7108:- Issue #15431: Add _freeze_importlib project to regenerate importlib.h on
7448:  (Modules/_freeze_importlib) to build Python/importlib.h.
8949:- Issue #13591: A bug in importlib has been fixed that caused import_module

Lib/importlib/_bootstrap.py
675:                # See # https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.load_module
```

* Issue: gh-103326

### CODE REVIEW

**Comment by sobolevn** [comment]
No idea what is going on:

```
Getting the list of files that have been added/changed ... 5 files
Fixing Python file whitespace ... 1 file:
  Tools/c-analyzer/cpython/_parser.py
```

I didn't touch any whitespaces 🤔

**Comment by sobolevn** [comment]
Oh, I see. This file contains tabs in https://github.com/python/cpython/blob/main/Tools/c-analyzer/cpython/_parser.py#L111-L116 and many other places. I don't think that it should be fixed, because tsv is named tab separated values for a reason.

I will open an new issue to ignore this file from this check.

**Comment by sobolevn** [comment]
I've opened https://github.com/python/cpython/issues/103334 to fix this problem. I don't think that it is a blocker.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -61,9 +61,6 @@ Python/traceback.c            @iritkatriel
 /Tools/build/parse_html5_entities.py   @ezio-melotti
 
 # Import (including importlib).
-# Ignoring importlib.h so as to not get flagged on
-# all pull requests that change the emitted
-# bytecode.
 **/*import*.c                 @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
```
Comment (hugovk) [inline .github/CODEOWNERS]: re: https://github.com/python/cpython/commit/811b2878dfc68dc3ecd81fb4b370c6e1f9ec69c2 / https://github.com/python/cpython/pull/5087:

Can we merge these two back together? (GitHub isn't let me do a multiline suggestion, so this is part 1)

```suggestion
**/*import*                   @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -61,9 +61,6 @@ Python/traceback.c            @iritkatriel
...
 **/*import*.py                @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
```
Comment (hugovk) [inline .github/CODEOWNERS]: part 2:

```suggestion
```

**Review by erlend-aasland — APPROVED** [review]

**Review by brettcannon — APPROVED** [review]

**Pull request merged**